manifests/plugin/openscap/params.pp

```diff
@@ -6,4 +6,7 @@
   $listen_on               = 'https'
   $openscap_send_log_file  = '/var/log/foreman-proxy/openscap-send.log'
   $spooldir                = '/var/spool/foreman-proxy/openscap'
+  $contentdir              = '/usr/share/foreman-proxy/openscap/content'
```

Comment:
`/usr/share/foreman-proxy/openscap/content`, `/usr/share/foreman-proxy/openscap/reports`, etc.
So I think this is right. Not sure why it is an FHS violation though.